src/hardware/serialport/softmodem.h

```diff
@@ -211,14 +211,14 @@ class CSerialModem : public CSerial {
 	std::unique_ptr<CFifo> tqueue;
 
 protected:
-	char cmdbuf[255] = {0};
+	char cmdbuf[353] = {0};
```

Comment:
Is there any relation to magic number "253" added to `DoCommand` method?

Reply:
The previous code didn't take the potential full length of a hostname into account (which is 253 characters). 

The AT command line can consist of a 99-character command sequence followed by D<phone/hostname>

So.. 99 + D + hostname can potentially hit 253 characters.

Reply:
Added space for a terminating null character (now 354), and added a comment describing this.